src/Modules/CalcDefence.lua

```diff
@@ -485,7 +485,11 @@ function calcs.defence(env, actor)
 		modDB:NewMod("SpellDodgeChance", "BASE", SpellSuppressionChance / 2, "Acrobatics")
 	end
 	
-	local totalSpellSuppressionChance = modDB:Override(nil, "SpellSuppressionChance") or modDB:Sum("BASE", nil, "SpellSuppressionChance")
+	
+	local weaponsCfg = {
```

Comment:
Might it be a good idea to extend this to most of the other defensive config options as well for future proofing.

Reply:
is there anything else that would be useful to include in the config ?